ios/web5wallet/web5wallet.entitlements

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-<plist version="1.0">
-<dict>
-	<key>com.apple.security.application-groups</key>
-	<array>
-		<string>group.website.tbd.wallet</string>
-	</array>
-</dict>
-</plist>
```

Comment:
This is completely fine, as we're not utilizing the application groups. But just wanted to call out that we will need this if we decided to bring back app extensions in the future.